src/app/static/src/app/store/plotData/plotData.ts

```diff
@@ -10,7 +10,13 @@ import {
 import { PlotName, plotNames } from "../plotSelections/plotSelections"
 import { mutations } from "./mutations"
 
-export type PlotData = CalibrateDataResponse["data"] | InputTimeSeriesData | CalibratePlotData | ComparisonPlotData | InputComparisonData | PopulationResponseData;
+export type PopulationPyramidData = {
+    data: PopulationResponseData;
+    nationalLevelData: PopulationResponseData;
+}
+
+export type PlotData = CalibrateDataResponse["data"] | InputTimeSeriesData | CalibratePlotData | ComparisonPlotData | InputComparisonData | PopulationPyramidData;
+export type SquarePlotData = Exclude<PlotData, PopulationPyramidData>;
```

Comment:
I think this is quite ugly... but not sure if there a better way to do this? Thought about committing the national data to some other place in state but that seemed quite ugly too. Or doing the national filtering later, but that it a big gross... very happy to hear if you have any good ideas, I am a little stumped

Reply:
I agree it is a bit awkward but I think it makes most sense to do the aggregating of national data and all other data in the same place as we have currently. Only other thing I can think is, as you said, making two state commits in the getPopulationFilteredData function and committing the national level data to a different place in state. The two data sets are so closely tied together though and are really only relevant to the plot so I'm inclined to say just go with the method you have of coupling the two data sets together in an object. We could rename the lowest "data" field to something like "subPlotData" to avoid the data.data repetition but not entirely necessary